roles/ruby_libvirt_gem/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: 'Install libvirt gem'
+  yum: name=tfm-rubygem-ruby-libvirt state=present
```

Comment:
I realise this was present before, but is this really needed? ```foreman-libvirt``` depends on this so it should be redundant.